code/modules/mob/living/silicon/robot/robot_defense.dm

```diff
@@ -94,6 +94,9 @@
 			locked = 0
 		else
 			to_chat(user, "<span class='warning'>The cover is already unlocked!</span>")
+
+		if(shell) //A warning to Traitors who may not know that emagging AI shells does not slave them.
```

Comment:
should this not be under the ```if(locked)``` part?
because:

The cover is already unlocked!
Spibble seems to be controlled remotely! Emagging the interface may not work as expected.

seems odd to me.